addons/mail/static/tests/message/message.test.js

```diff
@@ -1168,15 +1168,14 @@ test("not highlighting the message if not mentioning the current user inside the
 test("allow attachment delete on authored message", async () => {
     const pyEnv = await startServer();
     const channelId = pyEnv["discuss.channel"].create({ name: "test" });
+    const attachmentId = pyEnv["ir.attachment"].create({
+        mimetype: "image/jpeg",
+        name: "BLAH",
+        res_id: channelId,
+        res_model: "discuss.channel",
+    });
```

Comment:
Is there a specific reason why we need the attachment created separately in the tests now rather than as a command of message?